recipe/build.sh

```diff
@@ -10,3 +9,16 @@ bash configure  --prefix=$PREFIX $OPTS
 make
 make check
 make install
+
+if [ "$(uname)" == "Darwin" ]; then
+    # Based on
+    # https://github.com/Homebrew/homebrew-core/blob/master/Formula/emacs.rb
+    mv nextstep/Emacs.app $PREFIX/Emacs.app
```

Comment:
Interesting, what happens when you don't do this? Does the app still show up?

Another interesting question is where should the app go and should it have an extension? For instance, `python.app` seems to drop the `.` making it an ordinary directory, but is still in the top level of the prefix.